stores-api.yaml

```diff
@@ -1,95 +1,626 @@
+        hasShipping:
+          type: boolean
+        shopOwnerImage:
+          type: string
+        shopOwnerName:
+          type: string
+        id:
+          type: string
+        published:
+          type: boolean
+      example:
+        id: c3d6517f-b285-4a9e-b650-345c850bb01c
+        name:
+          de_DE: Agas Own 2
+        description:
+          de_DE: "\"Agas Own 2\" in Berlin Kreuzberg am Südstern. Erlebe einen Traum\
+            \ von Vielfalt aus dem Raum Asien im Bereich: Wohnaccessoires, ein Raum\
+            \ der Begegnung. \n\nHallo liebe Leute, Ihr seid auf mein bereits bestehendes\
+            \ Projekt/Ladengeschäft aufmerksam geworden. Ich betreibe seit über einem\
+            \ Jahr in Berlin-Kreuzberg am Südstern ein Ladengeschäft für Wohnaccessoires\
+            \ aus der Türkei, Thailand, Indonesien, Indien. Ich wohne selber seit\
+            \ einigen Jahren in der Körtestraße. Deshalb liegen mir als gebürtigem\
+            \ Berliner und Kreuzberger der Zusammenhalt im Kiez, der Austausch von\
+            \ Anliegen und die Mitmenschen, die ich in meinem Geschäft täglich antreffe,\
+            \ am Herzen."
+        address:
+          country: DE
+          streetName: Körtestraße
+          streetNumber: "34"
+          postalCode: "10967"
+          city: Berlin
+          state: ""
+        neighborhood:
+        - Kreuzberg
+        - Neukölln
+        - Körtekiez
+        geoLocation:
+          type: Point
+          coordinates:
+          - 13.408134
+          - 52.489908
+        profileImageURL: https://7cbf5172f31e91cb7a5d-d8e96be60ea1fd7ad98fb07a482142ee.ssl.cf3.rackcdn.com/store-front-X8l4AvUY.jpg
+        profileVideoURL: https://youtu.be/PBkb2mzbqKE
+        shopOwnerImage: https://drive.google.com/file/d/1Ls5Yjc760oEfnibC9Xv29OtALtmV_Ss4/view?usp=sharing
+        shopOwnerName: Oskar Fleer
+        phone: 030 98388469
+        homepage: https://www.startnext.com/agas-own-2
+    StoreDraft:
+      title: StoreDraft
+      type: object
+      properties:
+        address:
+          $ref: '#/components/schemas/Address'
+        contact:
+          type: string
+          deprecated: true
+        description:
+          $ref: '#/components/schemas/LocalizedString'
+        email:
+          type: string
+        geoLocation:
+          $ref: '#/components/schemas/GeoJSON'
+        homepage:
+          type: string
+        name:
+          $ref: '#/components/schemas/LocalizedString'
+        neighborhood:
+          type: array
+          items:
+            type: string
+        shopCategory:
+          type: string
+        phone:
+          type: string
+        contactWhatsapp:
+          type: string
+        contactInstagram:
+          type: string
+        contactFacebook:
+          type: string
+        contactFacetime:
+          type: string
+        contactSkype:
+          type: string
+        contactTwitter:
+          type: string
+        profileImageURL:
```

Comment:
Should be type file or something like that. It is an Image upload and not an url